scripts/prompt-release.js

```diff
@@ -39,21 +38,7 @@ const childProcess = require('@lerna/child-process')
       './packages/rum/src/apm-base.js'
     ])
 
-    /**
-     * Prompt for working directory clean
-     */
-    const answer = await PromptUtils.confirm(
-      'Are these changes ok (Check your working directory before continuing)'
-    )
-
-    if (!answer) {
-      log.error(
-        'release',
-        'Aborting the release process',
-        `Please run 'npm run release' again to start`
-      )
-      process.exit(1)
-    }
+    childProcess.execSync('git', ['add', './packages/rum/src/apm-base.js'])
```

Comment:
Not sure whether this is required

Reply:
Yes otherwise the file change wont be reflected in the currently created release before pushing the commit to GitHub and then later picked up by release.